Sources/StreamChatUI/Utils/StreamImageProcessor.swift

```diff
@@ -0,0 +1,52 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import Nuke
+import UIKit
+
+/// This class provides resizing operations for `UIImage`
+open class StreamImageProcessor {
```

Comment:
I think we should have the same pattern as the other image components:
- ImageProcessing
- NukeImageProcessing (Not StreamImageProcessing, since we are using Nuke here)

Reply:
As the functionalities provided by this class are very generic(i.e. cropping, scaling), didn't feel the need to create a protocol for this. However, I have renamed the class to `NukeImageProcessor` to better suit other namings. Let me know your thoughts about not having a protocol.

Reply:
I think for consistency and also for testability it is better we still introduce the protocol here 👍